docs_user/modules/openstack-mariadb_copy.adoc

```diff
@@ -55,6 +55,7 @@ endif::[]
 ifeval::["{build}" == "downstream"]
 MARIADB_IMAGE=registry.redhat.io/rhosp-dev-preview/openstack-mariadb-rhel9:18.0
 endif::[]
+STORAGE_CLASS_NAME=crc-csi-hostpath-provisioner
```

Comment:
ci issue seems relevant 

```
STORAGE_CLASS_NAME: unbound variable
```

https://logserver.rdoproject.org/16/316/01956cc0cd0432f59da6ca9dfa2c40e9c322a285/github-check/data-plane-adoption-osp-17-to-extracted-crc/7790de8/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_with_ceph_out_2024-03-01T07:44:20EST.log

Reply:
@marios All CI issues are addressed in latest PR.